system/security/securesocket/securesocket.hpp

```diff
@@ -45,6 +45,8 @@ enum SecureSocketType
 #define SSLogNormal 5
 #define SSLogMax    10
 
+#define SSL_ERROR_PORT_CHECK -9995
```

Comment:
Could get confusing to someone new reading the code using this error because it is named exactly like openssl library errors.  I'd consider putting some prefix on it to make it stand out from those.

Reply:
ok, renamed